packages/trader/src/App/Components/Form/InputField/input.jsx

```diff
@@ -35,7 +35,7 @@ const Input = ({
         if (current_focus === name) {
             ref.current.focus();
         }
-    }, [current_focus, name]);
```

Comment:
I think the ref should be created with `useRef()` for functional components and then it also shouldn't have to go into the dependency array here

Reply:
noted.